### DISCUSSION TITLE
turn off "on this page"

### DISCUSSION THREAD

**Discussion by graemeblair** [quarto-dev/quarto-cli#3419] [Q&A] [answered]
Hi - I saw it's possible to turn off sidebar and navbar. Is it possible to turn off for some pages the "on this page" right navigation? Useful for front pages for examples. Thank you!

**Comment by dragonstyle** [comment, marked as answer]
Yes! We call that the TOC so setting `toc: false` should do it...

**Comment by graemeblair** [comment]
Oh thank you so much! I couldn't find the term.
